ci/check-basics.py

```diff
@@ -79,6 +79,25 @@ def complain(message):
 
         mode = raw_diff.split()[1]
 
+        if path.startswith('tools/accuracy_checker/configs/') and path.endswith('.yml'):
+            if mode == '120000':
+                try:
+                    if (OMZ_ROOT / path).is_symlink():
```

Comment:
`OMZ_ROOT / path` is repeated 5 times here (and 1 more time in the existing code). It would be good to put that value in a variable.

Reply:
Done